### DISCUSSION TITLE
FOUC appears when loading the site

### DISCUSSION THREAD

**Discussion by AndrewShedov** [vercel/next.js#15257] [Help]
# Bug report

## Bug description

FOUC appears when loading the site, for styled components via `import styles from './About_ENG.module.scss'` and `import styles from './Menu.module.scss'`. 

## To Reproduce

1. Example - https://next-js-fouc.vercel.app/en (need to select the English version).
2. Repository - https://github.com/AndrewShedov/Next.js_FOUC

## Expected behavior

Loading the page without FOUC.

## Screenshots

Screenshots from PageSpeed ​​Insights.
![1](https://user-images.githubusercontent.com/47457457/87773113-5ac17c00-c82b-11ea-91f5-2d30b024fad4.jpg)![2](https://user-images.githubusercontent.com/47457457/87773591-0cf94380-c82c-11ea-968d-044024d3bf1c.jpg)

## System information

- OS: Windows 10 Enterprise 2015 LTSB x64
- Browsers: 
  Google Chrome Version 84.0.4147.89 (Official build) (64 bit)
  Microsoft Edge Version 84.0.522.40 (Official build) (64-bit)
  Mozilla Firefox (Official build) 78.0.2 (64-bit)
- Version of Next.js:  9.4.5-canary.36
- Version of Node.js:  14.5.0

## Additional context

1. FOUC does not appear if you include styles in the component itself via - "`<style jsx></style>`", example here https://next-js-fouc.vercel.app/fr (need to select the French version).
2. FOUC is not in Mozilla Firefox (Official build) 78.0.2 (64-bit).

**Comment by awareness481** [comment]
This is already addressed by Issue https://github.com/vercel/next.js/issues/12448
I believe there will be a fix for both dev and prod environments within the next version.

Edit: If this is a time-sensitive issue for you, I believe moving your css to your styles folder,
ex. an index.css with imports or all your css in the index.css file
will resolve this for now.
